guide/designing-products/usage-life-cycle.md

```diff
@@ -104,7 +107,7 @@ A user becomes interested in your product. Maybe they have heard about it severa
    layout = "float-left"
 %}
 
-The user tries the service for the first time. They made an intentional decision to move forward and expect to be satisfied. They are still not very familiar with exactly how your product works, and their expectations may not match what your product offers. Therefore it is very important to pay close attention to the design of the first-time use, also known as onboarding experience. Users should quickly and successfully achieve the task they have in mind while getting introduced to the basic concepts and best practices for using your product. This experience sets expectations for future interactions. If the first use is frustrating, users are less likely to return as they weight off whether the trouble is worth the benefit.
+The user decides to move forward with your product. They are still not familiar with exactly how it works, and their expectations may not match what your product offers. Therefore, it is crucial to tightly manage the [onboarding experience research]({{ '/guide/onboarding/introduction/' | relative_url }}). Your product should make it easy for users to quickly and successfully solve problems while being introduced to your product's basics and best practices. This sets expectations for future interactions. If the first use is frustrating, users are less likely to use your product again—or to recommend it.
```

Comment:
```suggestion
The user decides to move forward with your product. They are still not familiar with exactly how it works, and their expectations may not match what your product offers. Therefore, it is crucial to pay close attention to the design of the [onboarding experience]({{ '/guide/onboarding/introduction/' | relative_url }}). Your product should make it easy for users to quickly and successfully solve problems while being introduced to your product's basics and best practices. This sets expectations for future interactions. If the first use is frustrating, users are less likely to use your product again—or to recommend it.
```
Not sure what is meant by `tightly managing the onboarding experience research`. It's the design of the experience that needs to be well-crafted.